qa/zcash/performance-measurements.sh

```diff
@@ -2,7 +2,7 @@
 set -u
 
 
-DATADIR=./benchmark-datadir
+DATADIR=~/tempzcashstuff/ #/benchmark-datadir
```

Comment:
Looks like this is left-over from local testing. Needs reverting.

Reply:
oopss fixed